### ISSUE TITLE
Error response must be an array, not undefined.

### ISSUE THREAD

**Issue by griffind** [huafu/ember-dev-fixtures#1]
I am new to ember, so bear with me. I get the error above when changing the route, because dev-fixtures simulated response is undefined. If I go into ember inspector and grab an instance of the "people" route and do a .store.find('person') call, it fetches a response that looks like this:
`"[dev-fixtures] Simulating response:" Object { people: Array[2] }`

**app/routes/people.js**

``` javascript
import Ember from 'ember';

export default Ember.Route.extend({
  model: function () {
    return this.store.find('person');
  }
});
```

**app/models/person.js**

``` javascript
import DS from 'ember-data';

export default DS.Model.extend({
  name: DS.attr('string'),
  age: DS.attr('string'),
  sex: DS.attr('string')
});
```

**fixtures/person.js**

``` javascript
export default [
  {
    id: 1,
    name: 'joey tenny',
    age: '22',
    sex: 'male'
  },
  {
    id: 2,
    name: 'bob michael',
    age: '42',
    sex: 'female'
  }
];
```

Im sure I am doing something really obvious, but I had a hard time finding anyone else that has an issue with this add-on. Nothing is configured differently in my config/environment.js, or my app/adapters/application.js, or my Brocfile.js. I simply did an npm install and used the blueprint to generate the fixture. I have tried to put parenthesis around the keys, change my this.store.find call to findAll, and it was to no avail.

Any help is greatly appreciated, and let me know if you need any further information into what my current setup is.

**Comment by huafu** [comment]
Doing a new app from scratch now, and will do the same as you. It's possible that there is an issue, tho I am using this addon in a complex application and in http://huafu.github.io

**Comment by griffind** [comment]
@huafu I don't think it is an issue with the add-on because I have tried a similar structure with other fixture add-ons. Also, I appreciate you taking the time to look at this for me.

**Comment by huafu** [comment]
There is a very weird bug lol, working on fixing it now, but it's freaking weird lol

**Comment by huafu** [comment]
This is fixed in `0.0.7` but I still don't understand why

``` js
var responseFunction, response, data = {};
responseFunction = Ember.run.bind(null, function(){ return data; });
response = responseFunction();
console.log(response);
// logs `undefined` ?!?!
```
